clients/cloud/confluent-cli/README.md

```diff
@@ -31,7 +31,7 @@ $ ccloud topic create test1
 * `--property parse.key=true --property key.separator=,`: pass key and value, separated by a comma
 
 ```bash
-$ confluent local produce test1 --cloud --property parse.key=true --property key.separator=,
+$ confluent local produce -- test1 -- --cloud --property parse.key=true --property key.separator=,
```

Comment:
Is a double-dash needed on both sides of `test1` here? It doesn't look that way below for `confluent local consume`

Reply:
Nope, great catch!